packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -132,132 +151,182 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
-            src="https://s3.amazonaws.com/static.synapse.org/images/certified-microscope.svg"
-            className="SectionGraphic"
-            alt=""
-          />
-          <div className="SectionTextFlexContainer">
-            <h2>Get Credit For Your Research</h2>
-            <div>
-              <h3>Record Provenance</h3>
-              <p className="SectionBody">
-                Synapse provides tools to record and display Provenance for each
-                step of your analysis.
-              </p>
-            </div>
-            <div>
-              <h3>Mint a DOI</h3>
-              <p className="SectionBody">
-                A digital object identifier (DOI) provides a persistent and easy
-                way to reference your digital assets in publications &mdash;
-                including data, code, or analysis results.
-              </p>
-            </div>
-            <Button
-              variant="contained"
-              color="primary"
-              sx={buttonSx}
-              href={accessToken ? DASHBOARD_LINK : REGISTRATION_LINK}
-            >
-              {accessToken ? 'View Your Dashboard' : 'Get Started Now'}
-            </Button>
-          </div>
+        <div className="SynapseHomepage__Section__Centered">
+          <Typography
+            variant="h2"
+            sx={{ textAlign: 'center', paddingBottom: '30px' }}
+          >
+            Synapse In Action
+          </Typography>
+          <ProjectViewCarousel entityId={projectViewId} />
         </div>
       </div>
       <div className="SynapseHomepage__Section WhiteBackground">
-        <div className="FlexContainerReverse">
-          <img
-            src="https://s3.amazonaws.com/static.synapse.org/images/collaborate.svg"
-            className="SectionGraphic"
-            alt=""
-          />
-          <div className="SectionTextFlexContainer">
-            <h2>Collaborate</h2>
-            <div>
-              <h3>Communicate Your Findings</h3>
-              <p className="SectionBody">
-                Use the Synapse Wiki services to communicate your findings by
-                embedding rich content such as images, Tables, Provenance, and
-                LaTeX equations.
-              </p>
-            </div>
-            <div>
-              <h3>Share Your Research</h3>
-              <p className="SectionBody">
-                New Synapse Projects are private by default &mdash; share with
-                your colleagues, collaborators, or make your work public! Create
-                Synapse Teams to manage your collaborations.
-              </p>
-            </div>
-            <Button
-              variant="contained"
-              color="primary"
-              sx={buttonSx}
-              href={accessToken ? DASHBOARD_LINK : REGISTRATION_LINK}
-            >
-              {accessToken ? 'View Your Dashboard' : 'Get Started Now'}
-            </Button>
+        <div className="SynapseHomepage__Section__Centered">
```

Comment:
This is a terrible diff.  Best to look at the source file, specifically at the 2 new sections (no other changes were made)